src/UserSettings.hs

```diff
@@ -36,6 +36,10 @@ userFlavour = defaultFlavour { name = "user" } -- Modify other settings here.
 userPackages :: [Package]
 userPackages = []
 
+-- | Arguments to pass to builders. See @doc/user-settings.md@.
```

Comment:
Presumably, you meant to use `userArgs` when defining `userFlavour`?

See example here: https://github.com/snowleopard/hadrian/blob/master/doc/user-settings.md#command-line-arguments